lib/assets/stylesheets/govuk-frontend/all.js

```diff
@@ -0,0 +1,53 @@
+import { nodeListForEach } from './common'
```

Comment:
The javascript from Frontend is inside a folder called 'stylesheets'. Suggest moving the Frontend dependency into a vendor directory or something (assuming installing it via `npm install` is not possible)